catalog/ui/catalog-ui-search/src/main/webapp/component/annotation/annotation.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
✏️  It might be helpful for maintainability in the future if the `Note` vs `annotation` terminology is normalized between the Java and UI components

Reply:
Note is the terminology used for the back-end because it's generalized and notes could be attached to various metacard types. Annotation is used in the front-end as it pertains to annotating queries. However, if the team feels strongly about it, the terminology can be unified.